src/main/java/com/danielagapov/spawn/Services/User/UserService.java

```diff
@@ -46,17 +42,11 @@ public UserDTO getUserById(UUID id) {
     }
 
     public List<UserDTO> getUsersByTagId(UUID tagId) {
+        // TODO: change this logic later, once tags are setup.
         try {
-            List<UserFriendTagMapping> mappings = userTagMappingRepository.findByFriendTagId(tagId);
-            List<User> users = new ArrayList<>();
-            for (UserFriendTagMapping mapping : mappings) {
-                users.add(mapping.getUser2());
-            }
-            return UserMapper.toDTOList(users);
+            return UserMapper.toDTOList(repository.findAll());
         } catch (DataAccessException e) {
-            throw new DatabaseException("Failed to get users by tag ID: " + e.getMessage(), e);
-        } catch (Exception e) {
-            throw new ApplicationException("Unexpected error occurred while getting users by tag ID: " + e.getMessage(), e);
+            throw new RuntimeException("Error retrieving users", e);
```

Comment:
Could you please not undo the work Shane did in #57 in using the `DatabaseException` and `ApplicationException`

Reply:
Oops my bad, careless merging.

Reply:
Seems to be fixed now. Resolving